packages/controller-utils/src/util.ts

```diff
@@ -200,10 +200,10 @@ export function toHex(value: number | string | BN): Hex {
  * @param logError - Determines if the error should be logged.
  * @returns Promise resolving to the result of the async operation.
  */
-export async function safelyExecute(
-  operation: () => Promise<any>,
+export async function safelyExecute<T>(
```

Comment:
Nit: Could we rename the `T` generic params into a non-single letter variable e.g. `Result`?

Reply:
Done in https://github.com/MetaMask/core/pull/3629/commits/ac1629d42b54b1b76ba66c04321d19e0976c514d